.gitignore

```diff
@@ -10,6 +10,7 @@ capybara-*.html
 .rspec
 /db/*.pg
 /spec/tmp
+spec/.factories.rb.swp
```

Comment:
seems it is just your local issue with editor. I think it is shouldn't be in gitignore

Reply:
yeap